photon-lib/py/photonlibpy/multiTargetPNPResult.py

```diff
@@ -0,0 +1,51 @@
+from wpimath.geometry import Transform3d
+from photonlibpy.packet import Packet
+
+class PNPResult:
+
+    _NUM_BYTES_IN_FLOAT = 8
+    PACK_SIZE_BYTES = 1 + (_NUM_BYTES_IN_FLOAT * 7 * 2) + (_NUM_BYTES_IN_FLOAT* 3)
+
+    def __init__(self):
+        self.isPresent = False
+        self.best = Transform3d()
+        self.alt = Transform3d()
+        self.ambiguity = 0.0
+        self.bestReprojError = 0.0
+        self.altReprojError = 0.0
+
+    def createFromPacket(self, packet:Packet) -> Packet:
+        self.isPresent = packet.decodeBoolean()
+        self.best = packet.decodeTransform()
+        self.alt = packet.decodeTransform()
+        self.bestReprojError = packet.decodeDouble()
+        self.altReprojError = packet.decodeDouble()
+        self.ambiguity = packet.decodeDouble()
+        return packet
+
+    def __str__(self) -> str:
+        return f"PNPResult {{isPresent={self.isPresent}, best={self.best}, bestReprojError={self.bestReprojError}, alt={self.alt}, altReprojError={self.altReprojError}, ambiguity={self.ambiguity}}}"
```

Comment:
Consider using dataclasses rather than hand rolling this. (This goes for all the classes where you have a custom `__str__` which should be a `__repr__`.)